recipes/doxygen/all/patches/1.9.4-0001-enable-modern-compilers.patch

```diff
@@ -0,0 +1,25 @@
+From 5198966c8d5fec89116d025c74934ac03ea511fa Mon Sep 17 00:00:00 2001
+From: Dimitri van Heesch <doxygen@gmail.com>
```

Comment:
You should add a patch source with a link to the PR or commit this is from please